README.md

```diff
@@ -188,6 +188,44 @@ We can always use layer to search for data availability:
   const datesS1 = await layerS1.findDatesUTC(bbox, fromTime, toTime);
 ```
 
+
+## Cancelling requests
+
+You can also cancel requests when searching/fetching data.
+
+To do so a factory of tokens needs to be created and a token passed through a request configuration object. In this object can also be defined the amount of retires desired in the case the petition fails.
+
+In the example below, a cancelFactory is created, passing a token inside the configuration request object. The timeout will cancel the requests after 500 miliseconds, throwing an exception.
+
+This exception can be caught and identified by isCancelled, function that can be imported from the library.
+
+```typescript
+import { cancelFactory, isCancelled, RequestConfiguration } from '@sentinel-hub/sentinelhub-js';
+
+const source = CancelFactory.createSource();
```

Comment:
I must confess that I never understood why axios distinguishes between "source" and "token"... Do you think we could do something like this?
```
const cancelToken = CancelFactory.createToken();
...
const requestConfig : RequestConfiguration = {
  cancelToken: cancelToken,
  retries: 4, // request is tried up to 5 times
}
```

Reply:
I don't think so, since the source is the object needed to cancel the requests later on.
With the token alone you can't do that. It's not as convenient but I don't see any way to do it.

Reply:
Sorry if I wasn't clear... It is up to us to return whatever we want as "token". We can easily return "axios source" and then use `getToken()` on it when we need the token, internally - this is up to us. Or we could return an object which holds the source _and_ the token, if needed.

My concern here is that we might be leaking too much of axios implementation details to the outside.